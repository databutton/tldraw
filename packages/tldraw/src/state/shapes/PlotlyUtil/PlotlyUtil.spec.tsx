import { Plotly } from '..'

describe('Plotly shape', () => {
  it('Creates a shape', () => {
    expect(Plotly.create({ id: 'plotly' })).toMatchSnapshot('plotly')
  })
})
