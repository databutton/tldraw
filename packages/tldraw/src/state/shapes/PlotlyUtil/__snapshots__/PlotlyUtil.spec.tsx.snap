// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plotly shape Creates a shape: plotly 1`] = `
Object {
  "assetId": "assetId",
  "childIndex": 1,
  "id": "plotly",
  "name": "Plotly",
  "parentId": "page",
  "point": Array [
    0,
    0,
  ],
  "rotation": 0,
  "size": Array [
    1,
    1,
  ],
  "style": Object {
    "color": "black",
    "dash": "draw",
    "isFilled": true,
    "scale": 1,
    "size": "small",
  },
  "type": "plotly",
}
`;
